compiler/luci/pass/src/RemoveRedundantTransposePass.cpp

```diff
@@ -90,30 +90,26 @@ bool remove_consecutive_transpose_function(luci::CircleTranspose *target_node)
 
 namespace luci
 {
+
 /**
  *  BEFORE
  *         |
  *   [CircleNode]     [CircleConst]
- *    (main_node)      (main_perm)
- *         \               /
+ *         |           (pred_perm)
+ *          \              /
  *         [CircleTranspose]  [CircleConst]
- *            (pred_node)      (pred_perm)
+ *            (pred_node)     (target_perm)
```

Comment:
https://github.com/Samsung/ONE/blob/e2014ebf11b6e43c049ad7c46485a1734513e88e/compiler/luci/pass/src/RemoveRedundantTransposePass.cpp#L44-L46
If we change this comment from `perd_perm` to `target_perm`, then this should be changed to matching with comment.

Reply:
I'll post another PR as thic PR is to fix comments.

Reply:
#6153 posted